lib/vsc/install/shared_setup.py

```diff
@@ -704,7 +704,6 @@ class VscScanningLoader(ScanningLoader):
 
         TEST_LOADER_MODULE = __name__
 
-        # pylint: disable=arguments-differ
```

Comment:
this is still needed for the pylint on jenkins. Maybe we should set a tests_require for the pylint version?